src/ui/image_viewer.py

```
@@ -164,6 +203,74 @@ def _numpy_to_qimage(self, arr: np.ndarray) -> QImage:
+        new_arr = np.clip(new_arr, 0, 1)
+        # return the normalized array with edits
+        return new_arr
+
+
+    def _on_adjustment_changed(self, _value: int) -> None:
+        self._update_adjustment_labels()
+        self._show_current()
+
+
+    # Function to convert to 8 bits
+    def _ensure_uint8(self, arr: np.ndarray) -> np.ndarray:
```

Ruff (0.14.3)

Unused method argument: `arr`

(ARG002)